src/model/schema.js

```diff
@@ -510,38 +510,10 @@ export class SchemaItem {
 	 * @returns {Boolean} `true` if item has any registered matching path, `false` otherwise.
 	 */
 	_hasMatchingPath( type, checkPath, attribute ) {
```

Comment:
To make things less confusing we could rename `checkPath` to `pathToCheck` or `inputPath` or something. Right now when reading code it was difficult to keep in mind which paths were which.